.circleci/config.yml

```diff
@@ -52,6 +52,28 @@ jobs:
 
       - run: npm run publish-docs
 
+  publish_to_npm:
+    docker:
+      - image: circleci/node:8
+
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "package.json" }}
+          # fallback to using the latest cache if no exact match is found
+          - v1-dependencies-
+
+      - run: mv npmrc-env .npmrc
+
+      # Publish to NPM if there isn't already a published version with the current version tagged in git
+      - run: if [ $(npm show ilp-protocol-psk2 version) != $(npm ls --depth=-1 2>/dev/null | head -1 | cut -f 1 -d " " | cut -f 2 -d @) ] ; then npm publish ; fi
```

Comment:
These two `$()` expressions should probably be wrapped in quotes. Otherwise some really weird behavior will occur if either fails to return a value

Reply:
👍 you're the bash expert (note that I just copied it from the `ilp` module so it should probably be updated across all of our repos)